[WIP] Figure out the confict between LSPlant and LSPlt

JingMatrix opened the pull request:
As shown in #9, LSPlt is not working well for some devices (but not including mine).
**WRONG**: ~~It turning out that, by reverting the update of LSPlant, LSPlt works well on those devices.~~

In this pull-request, I am trying to figure out which commit of the LSPlant upstream is causing the problem.
Users of JingMatrix/LSPosed are welcomed to provide their test results on their devices to help me.

The goal is to ensure that the string `Dobby` doesn't appear in the log output of `adb logcat -s LSPlt LSPosed`, which means that **LSPlt has worked properly**.

**LSPlt doesn't work here neither**: ~~To start the test, please first confirm that LSPlst works with this build: https://github.com/JingMatrix/LSPosed/actions/runs/10641620655~~

Then please check if LSPlt works for CI builds of current pull-request and upload relevant logs here.

pershoot commented:
@JingMatrix Please let me know when you further hone it down and I will spare some time to test for further validation as I can reproduce it, on my device over here.  Thank you for your efforts!  It is appreciated.

JingMatrix commented:
@pershoot Yes, please try the latest build, and send me by emali: jingmatrix@gmail.com (these two files are too large)
1. Your `libart.so`, which is can pulled using `adb pull  /apex/com.android.art/lib64/libart.so`
2. The logs from your LSPosed manager

JingMatrix commented:
It turns out `PLSlt` is noly suitable for inline hooks in Android 15.
Hence, we put this pull-request aside and wait for Android 15 users to test out.

JingMatrix committed:
Manually set LD_LIBRARY_PATH for fexecve

Close #28 #1 as completed.
It would be better to induce this env from somewhere else if possible.

JingMatrix committed:
Update dependencies and fix CI

Remove ccache. Close #30 as completed.

JingMatrix committed:
Provide a build without PLT hook

JingMatrix committed:
Skip inline hook for function not loaded yet

Avoid using unallocated address, which might cause crashes
